NSubstitute.DbConnection/NSubstitute.DbConnection/MockQuery.cs

```diff
@@ -194,5 +244,22 @@ private static QueryInfo GetQueryInfo(IDbCommand mockCommand)
             };
             return queryInfo;
         }
+
+        private static void SetupOutputParams(IDbCommand mockCommand, Dictionary<string, QueryParameter> parameters)
+        {
+            for (var i = 0; i < mockCommand.Parameters.Count; i++)
+            {
+                var cmdParam = mockCommand.Parameters[i];
+                if (cmdParam is DbParameter dbParameter && dbParameter.Direction == ParameterDirection.Output)
+                {
+                    if (!parameters.TryGetValue(dbParameter.ParameterName, out var value))
+                    {
+                        throw new NotSupportedException($"Unmatched output parameter: '{mockCommand.CommandText}'");
```

Comment:
Ought this to be impossible, given that we've already matched up the command with the mock query?

Reply:
You're right, as I discovered when I attempted to write a test to cover this. Seems I forgot to remove it here